.github/workflows/ci.yml

```diff
@@ -7,17 +7,26 @@ on:
     branches:
       - '*'
 jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: 🧼 lint renovate config # Validates changes to renovate.json config file
+        uses: suzuki-shunsuke/github-action-renovate-config-validator@v0.1.2
```

Comment:
Should we use our own fork of this?  I don't know what good practice is here.

Reply:
generally don't have to fork a github action because you version lock it and github takes care of versioning....it's pretty much like maven artifacts

Reply:
It also appears in our documentation, so never mind.